### ISSUE TITLE
[Error View] Insufficient Color Contrast

### ISSUE THREAD

**Issue by emilyrohrbough** [cerner/terra-clinical#235]
# Issue Description
Accessibility testing resulted in the following:

- **Issue**: Ensure the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.
    - Message: Element has insufficient color contrast of 1.64
    - Impact: Serious
    - Contrast between body background (#fffff) and text font color (#c8cacb)

Note: The glyph also applies the color #c8cacb, however this does not result in an issue, likely due to its size in comparison to font text.

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [x] Bug
- [ ] Other

## Expected Behavior
Error View meets WCAG 2 AA contrast ratio thresholds.

## Current Behavior
The Error View text has insufficient color contrast of 1.64.

**Comment by neilpfeiffer** [comment]
## UX Design Input

Once https://github.com/cerner/terra-core/pull/1059 is merged and released, we will need to mark [terra-clinical-error-view](https://github.com/cerner/terra-clinical/tree/master/packages/terra-clinical-error-view) and [terra-clinical-no-data-view](https://github.com/cerner/terra-clinical/tree/master/packages/terra-clinical-no-data-view) for deprecation. 

Color changes if we need to support fixes:
name/heading(title): `color: #868a8c;`
description/subtext(message): `color: #1c1f21;`

**Comment by mjhenkes** [comment]
Closing as error view is now deprecated with this pr https://github.com/cerner/terra-clinical/pull/253.
